examples/lock-app/nxp/zap/lock-app.matter

```diff
@@ -2749,6 +2770,30 @@ endpoint 1 {
     handle command LockDoor;
     handle command UnlockDoor;
     handle command UnlockWithTimeout;
+    handle command SetWeekDaySchedule;
```

Comment:
should the featureMap be also modified? it seems it is 0x0001 (i.e. PIN) but it seems we want WDSCH as well and maybe others.

Reply:
I updated the files. Thank you